analysis/build.gradle

```diff
@@ -32,7 +33,7 @@ buildscript {
 
         mockito_core_version = '3.12.4'
         mockito_kotlin_version = '2.2.0'
-        mokk_version = '1.12.3'
+        mockk_version = '1.12.3'
```

Comment:
Is that on purpose?

Reply:
Yes, it is on purpose. It's for the Mock Kotlin dependency https://mockk.io/